test/index_test.js

```diff
@@ -382,8 +382,9 @@ tape.test('v5 compile', function(t) {
                         options.legCount = fixture.options.legCount;
                         options.classes = fixture.options.classes;
                     }
-
                     Object.keys(fixture.instructions).forEach((l) => {
+                        // ignore custom instructions that don't get compiled
+                        if (!fixture.step) return;
```

Comment:
What does this solve?

Reply:
this code was trying to test the new fixtures I added, but they don't get compiled by default, you have to specifically grab them by the right path. The fixtures I created don't have steps connected to them, so I just added this line so they don't get included